### DISCUSSION TITLE
Find and replace in open files

### DISCUSSION THREAD

**Discussion by kumar007git** [atom/atom#22318] [General]
How can I find and replace in open files? All the research I've done indicates there is no such functionality in Atom. That's a shame. that this functionality is still not implemented even after this many years. All other editors have this feature. Is there any other alternative to accomplish this?

**Comment by Tomdazy** [comment]
There is already an issue opened for that: https://github.com/atom/find-and-replace/issues/241
Check this out, I think it might help you : [find and replace in folder](https://github.com/trevdor/find-and-replace/commit/00b8d57be67bd37dd87d8934f302403434d191ef)

**Comment by kumar007git** [comment]
Thanks. I saw the previous issues related to this, but wanted to check if I missed its implementation. This issue has been around for many years, so why Atom Editor developers did not still implement this functionality? Is it that difficult?

I looked at the other thread link you shared, but it is too confusing for me.

**Reply by UziTech** [reply]
I'm not sure you understand how free open source software works. There is a small group of main Atom contributors who mainly act to triage bugs and maintain Atom as well as occasionally adding new features to Atom. About 80% of commits (especially new features) to any open source software are done by people not in that group.

If you want something to get done in free open source the best way to get is to submit a pull request with the change. Relying on the people tasked with fixing bugs and triaging issues on a software they don't get paid for won't get very far.

This issue has been open for so long because the number of people that find it useful is small enough to not include any person willing to put the work into actually coding it.
